pkg/controller/ingress/concurrency_watchdog.go

```diff
@@ -159,6 +159,7 @@ func (c *ConcurrencyWatchdog) tick(ctx context.Context) error {
 		list := &corev1.PodList{}
 		err := c.client.List(ctx, list, client.InNamespace(c.config.NS), client.MatchingLabels(target.LabelGetter.PodLabels()))
 		if err != nil {
+			c.logger.Error(err, "error listing pods")
 			retErr = multierror.Append(retErr, err)
```

Comment:
just noticing the error we return here isn't formatted.

should probably format the thing we are appending to give it context